src/kafunk/Utility/Faults.fs

```diff
@@ -148,7 +148,22 @@ module RetryPolicy =
   /// Returns an unbounded retry policy with a linearly increasing delay.
   let linearBounded (init:TimeSpan) (increment:TimeSpan) (attempts:int) = 
     linear init increment |> maxAttempts attempts
+    
+  let checkMax (s: TimeSpan) = 
+    let upper = TimeSpan.FromMilliseconds 2000000000
+    min upper s
+
+  let randomize r = 
+    let rand = System.Random()
+    let hi, lo = 1.0 + r, 1.0 - r
+    fun (s: TimeSpan) -> (float s.Milliseconds) * (rand.NextDouble() * (hi - lo) + lo) |> TimeSpan.FromMilliseconds
+
+  let exp (init: TimeSpan) (multiplier) : RetryPolicy = 
```

Comment:
Would you make `exp` decoupled from `randomize` and make a another one called `expRand`?